src/test/resources/squidProxy/squid.conf.no_authentication

```diff
@@ -0,0 +1,104 @@
+#
```

Comment:
these are just the things I used & didn't want to lose.  Took a while to get the right version & config to get  websocket support.